modules/websearch/lib/websearch_webinterface.py

```diff
@@ -279,6 +279,14 @@ def __call__(self, req, form):
                 navmenuid='search')
 
         from invenio.search_engine import record_exists, get_merged_recid
+
+        # Log page view to obelix
+        try:
```

Comment:
Please wrap in if CFG_OBELIX... so that statements would be executed only if Obelix is enabled.